integration/spark/src/main/spark3/java/io/openlineage/spark3/agent/lifecycle/plan/catalog/DatabricksDeltaHandler.java

```diff
@@ -0,0 +1,90 @@
+package io.openlineage.spark3.agent.lifecycle.plan.catalog;
+
+import io.openlineage.spark.agent.facets.TableProviderFacet;
+import io.openlineage.spark.agent.util.DatasetIdentifier;
+import io.openlineage.spark.agent.util.PathUtils;
+import java.lang.reflect.InvocationTargetException;
+import java.util.Arrays;
+import java.util.Map;
+import java.util.Optional;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.commons.lang3.reflect.MethodUtils;
+import org.apache.hadoop.fs.Path;
+import org.apache.spark.sql.SparkSession;
+import org.apache.spark.sql.catalyst.TableIdentifier;
+import org.apache.spark.sql.connector.catalog.Identifier;
+import org.apache.spark.sql.connector.catalog.TableCatalog;
+import scala.Option;
+
+@Slf4j
+public class DatabricksDeltaHandler implements CatalogHandler {
+  public boolean hasClasses() {
+    try {
+      DeltaHandler.class
+          .getClassLoader()
+          .loadClass("org.apache.spark.sql.delta.catalog.DeltaCatalog");
+      return true;
+    } catch (Exception e) {
+      // swallow- we don't care
+    }
+    return false;
+  }
+
+  @Override
+  public boolean isClass(TableCatalog tableCatalog) {
+    return tableCatalog
+        .getClass()
+        .getCanonicalName()
+        .equals("com.databricks.sql.transaction.tahoe.catalog.DeltaCatalog");
+  }
+
+  @Override
+  public DatasetIdentifier getDatasetIdentifier(
+      SparkSession session,
+      TableCatalog tableCatalog,
+      Identifier identifier,
+      Map<String, String> properties) {
+    // DeltaCatalog catalog = (DeltaCatalog) tableCatalog;
```

Comment:
can be removed

Reply:
Ah!  Good catch!  Thank you! It has been removed in the latest force pushed update.